src/schema/ecommerce/query_helpers.ts

```diff
@@ -0,0 +1,25 @@
+import gql from "lib/gql"
+
+export const RequestedFulfillmentFragment = gql`
```

Comment:
and include it at the top, that's a good idea 👍

Reply:
In [another PR](
https://github.com/artsy/metaphysics/pull/1239#discussion_r212623281) we went for an anonymous inline fragment, because then you can just interpolate it inline.